Soccar [UI]/src/nl/socnet/message/SpawnBallMessage.java

```diff
@@ -0,0 +1,49 @@
+package nl.socnet.message;
+
+import nl.soccar.library.enumeration.BallType;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.SpawnBallMessageHandler;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SPAWN_BALL_MESSAGE_ID, handler = SpawnBallMessageHandler.class)
+public final class SpawnBallMessage extends Message {
+
+    private final float x;
+    private final float y;
+    private final float angle;
+
+    private final BallType type;
+
+    public SpawnBallMessage(float x, float y, float angle, BallType type) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)